### DISCUSSION TITLE
Use media in Stiches

### DISCUSSION THREAD

**Discussion by ovinidev** [stitchesjs/stitches#966] [Help]
I tried to use media with @, but I couldn't
![2022-02-21_21-10](https://user-images.githubusercontent.com/62653539/155122487-a07f1886-06f9-4ad7-b69d-dca1f80c5f74.png)
![2022-02-21_21-10_1](https://user-images.githubusercontent.com/62653539/155122504-a7ff0891-0eed-4d4a-98c8-a0af4ea4400b.png)

**Comment by hadihallak** [comment]
Media should be outside the theme.
